docs/source/conf.py

```diff
@@ -11,9 +11,9 @@
 copyright = 'Documentation under CC-BY 4.0, Benjamin Worpitz, René Widera, Axel Huebl, Michael Bussmann'
 author = 'Benjamin Worpitz, René Widera, Axel Huebl, Michael Bussmann'
```

Comment:
We should probably update this line at some point...

Reply:
Yes, please do this!